app/features/hooks/useScroll.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
```

Comment:
I am using the hook to animate the Header component and make it more interesting to the eye